gensim/utils.py

```diff
@@ -1709,6 +1709,28 @@ def prune_vocab(vocab, min_reduce, trim_rule=None):
     return result
 
 
+def merge_dicts(dict1, dict2):
```

Comment:
IMO `merge_dicts` seems too general a name for an operation as specific as this, it's slightly confusing. What do you think about `aggregate_counts` or `merge_counts`?

Reply:
Agree, `merge_counts` sounds good